react-app/components/Gallery/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { useContext, memo } from 'react';
+import ImageGallery from 'react-image-gallery';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import 'react-image-gallery/styles/css/image-gallery.css';
+import { GalleryContext } from '@contexts/galleryContext';
+
+import './styles.css';
+// TODO: Remove this images ulr's when files come from backend
+const images = [
+  {
+    original: 'https://fastly.picsum.photos/id/1018/1000/600.jpg?hmac=8y6PgnvgTLEEW-118lVn6V6zPUVSN9JSi27GSpmGpdQ',
+    thumbnail: 'https://fastly.picsum.photos/id/1018/250/150.jpg?hmac=a9jeH9XHKHtbPkhvklrYWgcEOX9kfjewcCNGPOrbAs0'
+  },
+  {
+    original: 'https://picsum.photos/id/1015/1000/600/',
+    thumbnail: 'https://picsum.photos/id/1015/250/150/'
```

Comment:
where are going to get the thumbnails when uploading our own files?

Reply:
When files will be uploaded we need to save it as well

Reply:
But how do we generate them? Can we use the original image as the thumbnail? I'm not sure how that works

Reply:
We should use two images, can be the same but the dimensions will be affect. We can disccust that after